docs/getting-started.md

```diff
@@ -139,31 +159,41 @@ $ ../../gradlew run --args="-mode storage -action charge -amount 0 -to merchant1
 $ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
 ```
 
-## Store & retrieve data with transaction service
+## Set up database schema with transaction
+
+To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.
+
+```json
+{
+  "emoney.account": {
+    "transaction": true,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
+    }
+  }
+}
+```
 
-The previous application seems fine under ideal conditions, but it is problematic when some failure happens during its operation or when multiple operations occur at the same time because it is not transactional.
-For example, money transfer (pay) from `A's balance` to `B's balance` is not done atomically in the application, and there might be a case where only `A's balance` is decreased (and `B's balance` is not increased) if a failure happens right after the first `put` and some money will be lost.
+Before reapplying the schema, please drop the existing namespace first by issuing the following. 
 
-With the transaction capability of Scalar DB, we can make such operations to be executed with ACID properties.
-Before updating the code, we need to update the schema to make it transaction capable by adding `TRANSACTION` keyword in `CREATE TABLE`.
+* Download `scalar-schema-<vesrion>.jar` file from [scalardb release](https://github.com/scalar-labs/scalardb/releases)
```

Comment:
```suggestion
```

Reply:
Thank you for the suggestion.
Document updated with the above suggestion.